mne/dipole.py

```diff
@@ -43,4 +83,33 @@ def read_dip(fname, verbose=None):
     amplitude = data[:, 5]
     ori = data[:, 6:9]
     gof = data[:, 9]
-    return time, pos, amplitude, ori, gof
+    dipoles = dict(time=time, pos=pos, amplitude=amplitude, ori=ori,
+                   gof=gof, name=name)
```

Comment:
@agramfort this is what I changed `read_dipoles` to return (and `read_dip` splits it back out into a tuple), so I can just make it a class